libs/k8s-client/provisioning/local/kubernetes/main.tf

```diff
@@ -33,40 +32,28 @@ resource "kubernetes_secret" "k8s_client" {
   type = "kubernetes.io/service-account-token"
 }
 
-# this is namespace used by tests, can be completely purged by tests, except the role & role-binding
```

Comment:
Chápu proč to vzniklo. Ale v tuhle chvíli mi to přišlo zbytečný. https://github.com/keboola/platform-libraries/pull/95#discussion_r1178181624